cogs/wordfilter/wordfilter.py

```diff
@@ -489,7 +499,17 @@ async def checkWords(
                     blacklistedCmd = True
 
         try:
+            # records which words were used and how often
+            filterStats = await self.config.guild(msg.guild).get_attr(KEY_USAGE_STATS)()
+
+            for word in filteredWords:
+                timesMatched = len(re.findall(word, filteredMsg))
```

Comment:
As explained in side discussion, this detects more than what `_filterWord` does because there is no `\b`. A simple `\b` wrap should be sufficient to match its behavior with `filterWord`.

```suggestion
                timesMatched = len(re.findall(r"\b" + word + r"\b", filteredMsg))
```